boa/src/builtins/array/mod.rs

```diff
@@ -1204,6 +1204,11 @@ impl Array {
         // Static Methods
         make_builtin_fn(Self::is_array, "isArray", &array, 1, interpreter);
 
-        (Self::NAME, array)
+        let mut global = interpreter.global().as_object_mut().expect("Expect object");
+        global.borrow_mut().insert_property(
+            Self::NAME,
+            array,
+            Attribute::WRITABLE | Attribute::NON_ENUMERABLE | Attribute::CONFIGURABLE,
+        );
```

Comment:
```suggestion
        let mut global = interpreter.global().as_object_mut().expect("Expect object");
        global.borrow_mut().insert_property(
            Self::NAME,
            array,
            Attribute::WRITABLE | Attribute::NON_ENUMERABLE | Attribute::CONFIGURABLE,
        );
```
I think the first solution would be nicer, since we dont have to repeat the code and we dont have to do extra checks, so we should return `(&'static str, Value, Attribute)` we should only have to do:
```
fn init(...) -> (&'static str, Value, Attribute) {
 	// ...
	let attribute = Attribute::WRITABLE | Attribute::NON_ENUMERABLE | Attribute::CONFIGURABLE;
	(Self::NAME, value, attribute)
}
```